<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="receData">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>271</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>跳转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portSetting">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>串口配置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
